fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1038,10 +1363,10 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 			writeThread = createGenericThreadPool();
 			readThreads = createGenericThreadPool();
 		}
-		writeThread->addThread(new Writer(db, id, readIterPool), "fdb-rocksdb-wr");
+		writeThread->addThread(new Writer(db, id, readIterPool, perfContextMetrics), "fdb-rocksdb-wr");
 		TraceEvent("RocksDBReadThreads").detail("KnobRocksDBReadParallelism", SERVER_KNOBS->ROCKSDB_READ_PARALLELISM);
 		for (unsigned i = 0; i < SERVER_KNOBS->ROCKSDB_READ_PARALLELISM; ++i) {
-			readThreads->addThread(new Reader(db, readIterPool), "fdb-rocksdb-re");
+			readThreads->addThread(new Reader(db, i, readIterPool, perfContextMetrics), "fdb-rocksdb-re");
```

Comment:
I think the thread name size limit is 15, so you'd only have room for single digit `i` unless you shortened the name.

Reply:
That's a good point, another option is to shorten the prefix `fdb-rocksdb-re`, since we are distinguishing these reader threads, it might make sense to name them differently.

With that said, I don't have a strong preference on this, as the `nit` suggested.